xla/service/gpu/gpu_compiler_test.cc

```diff
@@ -970,6 +977,10 @@ using GpuCompilerPassTest = GpuCompilerTest;
 
 TEST_F(GpuCompilerPassTest,
        GpuCompilerRunsTritonGemmRewriterByDefaultFromAmpere) {
+  if (std::holds_alternative<se::RocmComputeCapability>(GpuComputeComp())) {
```

Comment:
I'm confused about this code. A few lines below there is an expectation that the triton gemm rewriter has run on rocm.

Can you please do two things:
- Explain why the rewriter is no longer needed. Did it ever run? What changed?
- Remove the `is_rocm` expecation below (lines 989-995).

Reply:
@dimitar-asenov 
> * Explain why the rewriter is no longer needed. Did it ever run? What changed?

The reason why we want, temporarily, to remove rewriter is that we have noticed significant performance drop when it is executed.
Performance drop is most probably related to the fact that, currently, triton gemm autotuner is not executed for rocm, 

Idea with this PR is to, temporarily, switch off triton gemm rewriter until implementation of rocm autotuner is finished, after that perform benchmarking, performance analysis and then switch it on again.